### PR TITLE
Implement Refactoring of Just Track- Moving away from Objective C Notation

### PR DESCRIPTION
**Pull request by danlages**
A general refactor to Just Track, moving away from the legacy Objective C Prefixes and Notation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,31 +178,59 @@ public class JEEventUser: NSObject, JEEvent {
...
+* [ ] ~~GoogleAnalyticsTraker~~ (not yet implemented, Google's pods can't be used as a dependency in a pod)
+* [ ] ~~TrakerFirebase~~ (not yet implemented, Google's pods can't be used as a dependency in a pod)
+
+## Upgrading to v4.0
+
+In version 4.0 Just Track has been refactoried to bring it up to date with current Swift standards. As such, Just Track no longer supports Objective C implementations. Please consider updating any applications that consume this feature.  As a result, the way events are named and must be called has been adjusted substatially. Please consider the following point when upgrading
+
+### Adopting Swift 
+
+This update to Just Track removes Objective C attributes and prefixes, modernising the implmenetation in line with Swift standards whilst also removing the Objective C Interopilbility.
```
Comment (antoniostrijdom) [inline README.md]:
```suggestion
This update to Just Track removes Objective C attributes and prefixes, modernising the implementation in line with Swift standards whilst also removing the Objective C interoperability. 
```

**Review by antoniostrijdom — CHANGES_REQUESTED** [review]
See comments.

File: README.md (hunk 1 above)
Comment (danlages) [reply README.md]: thanks man

File: README.md [hunk 2]
```diff
@@ -165,31 +178,59 @@ public class JEEventUser: NSObject, JEEvent {
...
+A Tracker is an object implementing the **Tracker** protocol and is loaded using: ```tracker.loadCustomTracker( ... )``` function. You can implement any tracker you want and **JustTrack** provides a few default trackers:
+
+* [x] TrackerConsole - print events to the system's console
+* [ ] FacebookTraker (not yet implemented)
+* [ ] ~~GoogleAnalyticsTraker~~ (not yet implemented, Google's pods can't be used as a dependency in a pod)
+* [ ] ~~TrakerFirebase~~ (not yet implemented, Google's pods can't be used as a dependency in a pod)
+
+## Upgrading to v4.0
+
+In version 4.0 Just Track has been refactoried to bring it up to date with current Swift standards. As such, Just Track no longer supports Objective C implementations. Please consider updating any applications that consume this feature.  As a result, the way events are named and must be called has been adjusted substatially. Please consider the following point when upgrading
```
Comment (albertodebortoli) [inline README.md]: refactoried -> refactored

Comment (albertodebortoli) [reply README.md]: Objective C -> Objective-C

File: README.md (hunk 1 above)
Comment (albertodebortoli) [inline README.md]: Objective C -> Objective-C (twice)

Comment (albertodebortoli) [reply README.md]: implmenetation -> implementation

Comment (albertodebortoli) [reply README.md]: Interopilbility -> interoperability

File: README.md [hunk 3]
```diff
@@ -165,31 +178,59 @@ public class JEEventUser: NSObject, JEEvent {
...
+```
+JEEventUser
+```
+
+will now adopt the naming scheme:
+
+```
+EventUser
+```
+The effects of this change on your pre-defined events can be determined within the generated TrackingEvents.Swift file.
```
Comment (albertodebortoli) [inline README.md]: .Swift -> .swift

**Review by albertodebortoli — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (danlages) [reply README.md]: Thanks!

**Review by antoniostrijdom — APPROVED** [review]

**Pull request merged**